src/components/Modals/CourseMenu.vue

```diff
@@ -35,15 +28,23 @@
           <button
             v-for="(color, index) in colors"
             :key="index"
-            class="courseMenu-section full-opacity-on-hover"
-            @click="colorCourse(color)"
+            class="courseMenu-color full-opacity-on-hover"
+            @click="openEditColorModal(color.hex)"
+            @mouseover="setDisplayColorTooltip(true, color.text)"
+            @mouseleave="setDisplayColorTooltip(false, color.text)"
           >
             <div class="courseMenu-left">
-              <div
-                class="courseMenu-icon courseMenu-icon--color"
-                :style="{ backgroundColor: color.hex }"
-              ></div>
-              <span class="courseMenu-text">{{ color.text }}</span>
+              <div class="courseMenu-color--icon" :style="{ backgroundColor: color.hex }">
+                <img
+                  v-if="`#${courseColor}` === color.hex"
```

Comment:
Instead of checking for hex value here, could we check for color name? This would resolve the issue with checkmarks not showing for existing users, and since the hex value changes are not really noticeable, I think it is ok to "lie" to the user and show that they are using the same color.

Reply:
I looked into this and I don't know if we can match with color name at all, since we are only storing the hex value in each course object. For example:

`code: "CS 1110"
color: "FF3B30"
creditRange: (2) [4, 4]
credits: 4
crseId: 358526
lastRoster: "FA21"
name: "Introduction to Computing Using Python"
semesters: (3) ['Fall', 'Spring', 'Summer']
uniqueID: 200`

Reply:
> I'm noticing the color tooltip names are covered by the cursor when hovering, at least on my windows machine, which makes it hard to read the names (especially the 2nd rows Cornell themed ones), or hover over other colors (if hovering over one in the top row, the one directly below gets covered by the tooltip). Could they be adjusted at all?
I adjusted this issue.

Reply:
> I looked into this and I don't know if we can match with color name at all, since we are only storing the hex value in each course object. 

Shoot yeah I think you're right. If we really want to, I think instead we could run a script on the Firestore (at any time) that would change a user's hex codes from the old values to the new values in the semesters collection. Not sure if that is worth it but it would probably resolve the arrow issue. This is definitely good for now though!